flatdata-rs/lib/src/structs.rs

```diff
@@ -140,6 +144,6 @@ mod test {
 
     #[test]
     fn test_range() {
-        assert_eq!(<R as Struct>::IS_OVERLAPPING_WITH_NEXT, true);
+        const _: [(); 0 - !{ <R as Struct>::IS_OVERLAPPING_WITH_NEXT } as usize] = [];
```

Comment:
What is this? :-)

Reply:
This is static assert: IS_OVERLAPPING_WITH_NEXT is actually just a constant and clippy complains that there is nothing to test. I added an allow directive and reverted it back.